changes/ocd-3824.sql

```diff
@@ -0,0 +1,24 @@
+-- change email address for jira api key
+UPDATE openchpl.api_key
+SET email = 'CSrinadhu@ainq.com',
+name_organization = 'Ai Jira Team'
+WHERE api_key_id = 605;
+
+-- delete data from deprecated response field table
+UPDATE openchpl.deprecated_response_field
+SET deleted = TRUE
+WHERE removal_date = '2022-03-01';
+
+-- delete views 
+DROP VIEW IF EXISTS openchpl.acb_developer_transparency_mappings;
+DROP VIEW IF EXISTS openchpl.developers_with_attestations;
+DROP VIEW IF EXISTS openchpl.certified_product_details;
+
+-- delete tables
+DROP TABLE IF EXISTS openchpl.acb_vendor_map;
+ALTER TABLE openchpl.pending_certified_product DROP COLUMN IF EXISTS vendor_transparency_attestation;
+
+-- delete columns
+-- dropping the type by both names (can remove first line below after OCD-3717 goes to STG)
+DROP TYPE IF EXISTS openchpl.attestation;
```

Comment:
Should this line be removed?

Reply:
I did put that comment in there. Maybe it should say when 3717 goes to PROD? At this point I don't think I would know which order the ocd sql files would be executed.

Reply:
Maybe we leave this here as is, and evaluate based on when 3717 goes to PROD and whether this one goes with it or not